WcaOnRails/app/views/polls/edit.html.erb

```diff
@@ -54,6 +54,7 @@
       <div class="col-sm-offset-2 col-sm-10">
         <%= f.button :submit, class: "btn-primary" %>
         <% if !@poll.confirmed %>
+          <%= link_to button_tag('Preview', type: "button", class: "btn btn-info"), polls_preview_path %>
```

Comment:
Changed to the alert at the beginning. I made the link have target _blank so you don't lose risking the unsaved work.